src/me/StevenLawson/TotalFreedomMod/Commands/Command_trail.java

```diff
@@ -18,16 +18,42 @@
 import org.bukkit.plugin.RegisteredListener;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off]")
+@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off | glass]")
```

Comment:
This should be updated in the description as it will not always be "Pretty Rainbow Trails" anymore.